cli/src/cpp/utils/participant_subelement_parser.cpp

```diff
@@ -176,7 +176,47 @@ void participant_subelement_parser(
     }
     else if (element == NAME_SUBELEMENT)
     {
-        std::cout << "Participant name configuration not yet supported" << std::endl;
+        // Check help value
+        print_usage = check_help(values);
+        // Not keyed element
+        print_usage = print_usage || !check_keyed(false, keyed, message.str());
+        // Final element
+        print_usage = print_usage || !check_final_element(true, subelement, message.str());
+        // SET command requires one argument
+        print_usage = print_usage || (CommonCommands::SET == command && !check_command_arguments(command, 1,
+                values.size(), message.str(), true));
+        // PRINT and CLEAR require no argument
+        print_usage = print_usage || ((CommonCommands::CLEAR == command || CommonCommands::PRINT == command) &&
+                check_command_arguments(command, 0, values.size(), message.str(), true));
```

Comment:
```suggestion
                !check_command_arguments(command, 0, values.size(), message.str(), true));
```

Reply:
Good catch. As the print and clear parsers are not implemented, this is not yet accessible and testable.